### DISCUSSION TITLE
How do I change the Ant Media Server UDP port range?

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#4944] [FAQs] [answered]
According to the documentation, the default UDP port range is 50000-60000. How can we change the port range if we don't want to open such a large range?

**Comment by tremzo** [comment, marked as answer]
You can change the UDP port range from the applications properties file located at 

```
webapps/[application]/WEB-INF/red5-web.properties
```

First stop the server:

```
sudo service antmedia stop
```

Open your application’s properties file and change the following settings to add min/max port values:

```
Min port range value
settings.webrtc.portRangeMin=50000

Max port range value
settings.webrtc.portRangeMax=51000
```

Save the file and then restart Ant Media Server:

```
sudo service antmedia start
```

In this example, Ant Media Server uses ports between 50000 and 51000 (1000 ports) for WebRTC connections. 

**Please pay particular attention to the range used because it also limits the number of concurrent publishers and players. So assume that it uses 2 ports for each WebRTC connection, having 1000 ports open limits the server to 500 connections.**

**Reply by yashtandon113** [reply]
Hi All,
Let me give an update on this, starting from Ant Media Sever v2.6.2, the application settings be it in Standalone Mode or Cluster mode can be changed directly from the Web Panel without having to edit the application configuration file red5-web.properties and restarting the server.

For more details check the application configuration document https://antmedia.io/docs/guides/configuration-and-testing/ams-application-configuration/

Thank you.
